ext/standard/config.m4

```diff
@@ -13,14 +13,14 @@ int main(int argc, char **argv)
 	char *filename = tmpnam(NULL);
 	char buffer[64];
 	int result = 0;
-	
+
```

Comment:
Sorry for the whitespace changes, I haven't figured out how to tell Atom to ignore whitespace yet.